document/src/tests/documentupdatetestcase.cpp

```diff
@@ -746,8 +641,8 @@ DocumentUpdateTest::testUpdateWeightedSetEmptyParamValue()
 
     // Verify that the field was set in the document.
     auto fval1(doc->getAs<WeightedSetFieldValue>(field));
-    CPPUNIT_ASSERT(fval1.get());
-    CPPUNIT_ASSERT_EQUAL((size_t) 0, fval1->size());
+    EXPECT_TRUE(fval1.get());
```

Comment:
Consider `ASSERT_TRUE`, is dereferenced in the following expectation

Reply:
Fixed